scripts/ubuntu20_04_build_setup.sh

```diff
@@ -0,0 +1,23 @@
+
+echo "Install dependencies"
+apt update
+apt install -y git build-essential python3.8 python3-pip pkg-config mesa-common-dev libglu1-mesa-dev wget libpcsclite-dev libpcre3-dev libssl-dev libpulse-mainloop-glib0 libxkbcommon-x11-dev extra-cmake-modules cmake
+
+echo "Install QT"
+pip install -U pip
+pip install aqtinstall
+aqt install-qt linux desktop 5.14.2 gcc_64 -m qtwebengine qtlottie
+
+
+echo "Install GoLang"
+export GOLANG_SHA256="d2c095c95f63c2a3ef961000e0ecb9d81d5c68b6ece176e2a8a2db82dc02931c"
+export GOLANG_TARBALL="go1.17.12.linux-amd64.tar.gz"
+wget -q "https://dl.google.com/go/${GOLANG_TARBALL}"
+echo "${GOLANG_SHA256} ${GOLANG_TARBALL}" | sha256sum -c
+tar -C /usr/local -xzf "${GOLANG_TARBALL}"
+rm "${GOLANG_TARBALL}"
+ln -s /usr/local/go/bin/go /usr/local/bin
+
+echo "Export env var"
+export QTDIR=/5.14.2/gcc_64
```

Comment:
Please do NOT install things directly in host root like that. The standard location for packages like that should be `/opt`.

Preferably, it should be the same as on the Docker container, so `/opt/qt/5.14.0/gcc_64`.

Reply:
it will now be installed in /opt/qt but version is 5.14.2 and not 5.14.0